.github/workflows/build.yml

```diff
@@ -25,3 +25,18 @@ jobs:
           name: security-profiles-operator
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
       - run: make nix nix-arm64
+
+  image:
+    runs-on: ubuntu-latest
+    env:
+      USER: security-profiles-operator+github
+      IMAGE: quay.io/security-profiles-operator/build
+      TOKEN: ${{ secrets.QUAY_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - run: sudo podman build hack/build-image -t $IMAGE
+      - name: Push the image
+        if: github.ref == 'refs/heads/master'
+        run: |
+          sudo podman login -u $USER -p $TOKEN
+          sudo podman push $IMAGE
```

Comment:
This looks fine; though we could use the buildah action instead https://github.com/marketplace/actions/buildah-build

then the push to registry action https://github.com/redhat-actions/push-to-registry

Reply:
Sounds good, using them now.